css/components/card.css

```diff
@@ -0,0 +1,127 @@
+/* Styling required for making basic card structure */
+.card{
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+    width:22rem;
+    height: 25rem;
+    border-radius: 0.25rem;
+    box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%),
+    0 1px 1px 0 rgb(0 0 0 / 14%),
+    0 1px 3px 0 rgb(0 0 0 / 12%);
+}
+
+/* Styling required for making card horizontal */
+.card_horizontal{
+    flex-direction: row;
+    flex-wrap: wrap;
+    height: 10rem;
+
+}
+
+/* Divided Card into two parts:-1.)Content 2.)Actions
+Content contains:- Image,Header and Content(Body)
+Actions contains:-Button and Icons*/
+.card_content{
+    flex-basis: 90%;
+    border-radius: 5%;
+}
+.card_content:hover{
+    background-color: var(--grey_background);
+    border-radius: 0;
+}
+
+/* For horizontal cards*/
+.card_content_horizontal{
+    display: flex;
+    flex-basis: 100%;
+}
+
+
+.card_content_image{
+    width: 100%;
+    border-radius: 0.25rem;
+}
+
+.card_content_image_horizontal {
+    width: 40%;
+    border-radius: 0.25rem;
+}
+
+.card_content_header{
+    padding: 5px;
+}
+.card_content_header_horizontal{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 0.625rem;
+
+}
+.primary_header {
+    font-weight: bold;
+    font-size: large;
+}
+
+.secondary_header {
+    font-weight: lighter;
+    color: var(--grey-color-300);
+    font-size: small;
+}
+
+.card_content_body{
+    padding: 5px;
+}
+
+.card_actions{
+    flex-basis: 10%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+    padding: 0 0.625rem 5px 0.625rem;
+}
+.card_actions_horizontal{
+    display: flex;
+    flex-basis: 100%;
+}
+
+.card_button_action{
+    display: flex;
+}
+.card_icon_action{
+    margin-left: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 0.5rem;
+}
+
+.btn_card{
+    border: none;
+    background-color: transparent;
+    font-weight:500;
+    color: var(--primary-700);;
```

Comment:
remove the extra semicolon from the end of the line.

Reply:
done